provision/acc_provision/flavors.yaml

```diff
@@ -3355,3 +3355,11 @@ flavors:
     status: Experimental
     hidden: true
     order: 43
+  calico:
```

Comment:
Since we are tying this to a specific version of the operator (3.23.2 ?), we should have a flavor specific to that version.